router/deployment.yaml

```diff
@@ -29,6 +29,34 @@ spec:
 ---
 apiVersion: v1
 kind: Service
+metadata:
+  name: batch
+  labels:
+    app: batch
+spec:
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 5000
+  selector:
+    app: batch
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: batch2
+  labels:
+    app: batch2
+spec:
+  ports:
```

Comment:
Do these need to get deleted from the respective batch and batch2 directories in deployment.yaml?

Reply:
We currently don't do that (they are duplicated).  I admit this probably isn't ideal.  We'd probably want router to depend on the service deploys, but then you need to deploy all the services to deploy the router (it does DNS lookups up front and fails if any of the DNS looks fail).

Alternatively router could have all the services, and the individual service deploys depend on the router.  That's annoying because the config for each service is split in two places, but that's probably best.  I'll PR this separately.